docs/HACKING.md

```diff
@@ -6,21 +6,26 @@
 - In case hostmounted volume is used, the directory on the host must have 777 permissions and the following selinux labels (TODO).
 - In case secure cluster is used, the certificates must be pre-generated and uploaded to the secret `<elasticsearch_cluster_name>-certs`
 
-## Kubernetes
+## Kubernetes (Obsolete)
 
-Make sure certificates are pre-generated and deployed as secret.
-Upload the Custom Resource Definition to your Kubernetes cluster:
-
-    $ kubectl create -f deploy/crd.yaml
+Make sure certificates are pre-generated and deployed as secret, e.g., `make deploy-example-secret`.
 
 Deploy the required roles to the cluster:
 
-    $ kubectl create -f deploy/rbac.yaml
+    $ kubectl create ns openshift-logging
+    $ kubectl apply -f manifests/01-service-account.yaml 
+    $ kubectl apply -f manifests/02-role.yaml 
+    $ kubectl apply -f manifests/03-role-bindings.yaml
```

Comment:
i actually don't think these files have been kept up to date. i plan on removing them as part of a pr, we can probably remove references to them completely.